README.md

```diff
@@ -21,11 +21,24 @@ https://github.com/Automattic/woocommerce-payments/blob/trunk/docker/README.md
 
 Install the following plugins:
 - WooCommerce
+## Test account setup
 
 For setting up a test account follow [these instructions](https://docs.woocommerce.com/document/payments/testing/dev-mode/).
 
 You will need a externally accessible URL to set up the plugin. You can use ngrok for this.
 
+```ngrok http 8082```
+
+Alternatively you can use [Jurassic-tube](https://fieldguide.automattic.com/jurassic-tube/) developed internally for a permanent url. (works with only docker setup)
```

Comment:
I don't think it's useful to mention that link to private site as this is a public repo.
Also, Jurassic Tube is only for Automattic, so maybe no need to mention in readme.

Reply:
Ah! it makes sense, maybe we can keep the ngrock script, it will save some time googling.

This [fieldguide doc](https://fieldguide.automattic.com/jurassic-tube/) on jurassic tube looks like a good place to mention the new script.

Reply:
Made changes in commit [501ee6c](https://github.com/Automattic/woocommerce-payments/pull/1578/commits/501ee6c0ed5aa8973a37365074125c84ae065acd)

Reply:
Yeah. Also, in developing wcpay FG too.

Reply:
Once merged, I'll publish the changes I've made in jurassic tube and wcpay dev FG docs. Also I'll keep @achyuthajoy in loop regarding their PR https://github.com/Automattic/woocommerce-payments/pull/1591

Reply:
I have updated and published the jurassic tube and wcpay dev FG docs.